src/riak_dt.proto

```diff
@@ -251,4 +267,6 @@ message DtUpdateResp {
     optional sint64   counter_value = 3;
     repeated bytes    set_value     = 4;
     repeated MapEntry map_value     = 5;
-}
\ No newline at end of file
+    repeated bytes    gset_value    = 6;
```

Comment:
I will argue that this is also unnecessary. When making an update, the sender knows if they sent a `SetOp` or `GSetOp`. `set_value` is the only field needed.